Set NO_CMAKE_FIND_ROOT_PATH

The truetype fonts are provided locally with an absolute path to find them.

Without NO_CMAKE_FIND_ROOT_PATH, they cannot be found because the 
CMAKE_PREFIX_PATH is applied to the suggested path.

Upstream-Status: Pending

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

diff --git a/3rdparty/nanogui/CMakeLists.txt b/3rdparty/nanogui/CMakeLists.txt
index fb22011d..2c966b73 100644
--- a/3rdparty/nanogui/CMakeLists.txt
+++ b/3rdparty/nanogui/CMakeLists.txt
@@ -260,9 +260,9 @@ endif()
 # * Roboto-Bold.ttf,  Roboto-Regular.ttf => apt install fonts-roboto-fontface
 # * entypo+ is a custom font from: https://github.com/svenevs/nanogui-entypo
 #   (it is NOT the same one as provided by the entypo debian package)
-find_file(ENTYPO_TTF_FILE           entypo.ttf          PATHS "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED)
-find_file(ROBOTO_BOLD_TTF_FILE      Roboto-Bold.ttf     PATHS /usr/share/fonts/truetype/roboto-fontface/roboto/ "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED)
-find_file(ROBOTO_REGULAR_TTF_FILE   Roboto-Regular.ttf  PATHS /usr/share/fonts/truetype/roboto-fontface/roboto/ "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED)
+find_file(ENTYPO_TTF_FILE           entypo.ttf          PATHS "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED NO_CMAKE_FIND_ROOT_PATH)
+find_file(ROBOTO_BOLD_TTF_FILE      Roboto-Bold.ttf     PATHS "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED NO_CMAKE_FIND_ROOT_PATH)
+find_file(ROBOTO_REGULAR_TTF_FILE   Roboto-Regular.ttf  PATHS "${CMAKE_CURRENT_SOURCE_DIR}/resources" REQUIRED NO_CMAKE_FIND_ROOT_PATH)
 
 # Glob up resource files
 set(resources ${ENTYPO_TTF_FILE} ${ROBOTO_BOLD_TTF_FILE} ${ROBOTO_REGULAR_TTF_FILE})
